tracer/test/Datadog.Trace.Tests/Configuration/DynamicConfigurationTests.cs

```diff
@@ -87,25 +87,38 @@ public void DoesNotOverrideDirectLogsTags()
             TracerManager.Instance.DirectLogSubmission.Formatter.Tags.Should().Be("key1:value1");
         }
 
+        [Fact]
+        public void EnableTracing()
```

Comment:
Do we have any end-to-end tests in this repo at the moment? If this passes the appropriate system-test, then I don't think it's blocking the PR from merging, but it would certainly be a nice-to-have and could be a follow-up PR

Reply:
Good question! I'll look into this.